corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -638,6 +638,31 @@ def test_fetch_missing_media(self):
         content = media.fetch_attachment(list(media.blobs.keys())[0])
         self.assertEqual(data, content)
 
+    def test_fetch_missing_media_already_exists(self):
```

Comment:
some comments in this test would be useful to make it easier to understand what different steps are for